RELEASE_NOTES.md

```diff
@@ -1,15 +1,10 @@
 ## Enhancements
 
-- [KOGITO-3270](https://issues.redhat.com/browse/KOGITO-3270) - Operator should support ConfigMap injecting in KogitoServices
-- [KOGITO-2905](https://issues.redhat.com/browse/KOGITO-2905) - Change in image definition in the CRDs
-- [KOGITO-3039](https://issues.redhat.com/browse/KOGITO-3039) - Agnostic infrastructure layer configuration on Operator
-- [KOGITO-2113](https://issues.redhat.com/browse/KOGITO-2113) - Simplify binary build deployment
+- [KOGITO-3094](Provide data-index-infinispan/mongodb as independent images) - Provide data-index-infinispan/mongodb as independent images
+     - To switch to Mongodb persistence provider just provide the `quay.io/kiegroup/kogito-data-index:1.0.0-snapshot` on the Kogito CR
```

Comment:
```suggestion
     - To switch to Mongodb persistence provider just provide the `quay.io/kiegroup/kogito-data-index-mongodb:1.0.0-snapshot` on the Kogito CR
```
?

Reply:
and provide MongoDB env var ?

Reply:
I believe we need to write specific documentation of how to connect the Data Index to MongoDB using java properties via `spec.config` or `spec.propertiesConfigMap` parameter.

Reply:
I'll add it the docs, just trying to make it work on ocp yet, having hard times to find a operator easy to use, the only one for now is the percona mongodb community operator.